app/src/main/java/com/github/sdp_begreen/begreen/activities/MainActivity.kt

```diff
@@ -130,6 +135,32 @@ class MainActivity : AppCompatActivity() {
             R.id.mainNavDrawFollowers -> {
                 replaceFragInMainContainer(FollowersFragment())
             }
+            R.id.mainNavDrawUserList -> {
+               val photo = Photo("erfs", ParcelableDate(Date()),User(0, "Lui",0), "Profile")
```

Comment:
This list seems to be temporary (it will be removed afterward). It can be cleaner to mark the temporary code with a // TODO so we don't forget to change it. I noticed that you did it for some methods.

Reply:
Oh yeah I removed the comments and the TODO by error during this PR. Thanks for pointing that out!